SeaPublicWebsite/Startup.cs

```diff
@@ -117,10 +120,16 @@ private void ConfigureCookieService(IServiceCollection services)
             services.Configure<CookieServiceConfiguration>(
                 configuration.GetSection(CookieServiceConfiguration.ConfigSection));
             // Change the default antiforgery cookie name so it doesn't include Asp.Net for security reasons
-            services.AddAntiforgery(options => options.Cookie.Name = "Antiforgery");
+            services.AddAntiforgery(ConfigureAntiForgeryCookieOptions);
             services.AddScoped<CookieService, CookieService>();
         }
 
+        private void ConfigureAntiForgeryCookieOptions(AntiforgeryOptions options)
+        {
+            options.Cookie.Name = "Antiforgery";
+            options.Cookie.SecurePolicy = webHostEnvironment.IsDevelopment() ? CookieSecurePolicy.None : CookieSecurePolicy.Always;
+        }
+        

```

Comment:
Turns out HER has this exact change and it's breaking the health check since the health check is apparently over HTTP, not HTTPS. We'll need to add in an exception to the health check endpoint that it doesn't use an anti forgery cookie.

Reply:
Sounds good, thanks for checking